docs/minio-client-complete-guide.md

```diff
@@ -362,6 +363,14 @@ FLAGS:
 mc cp myobject.txt play/mybucket
 myobject.txt:    14 B / 14 B  ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  100.00 % 41 B/s 0
 ```
+
+*Example: Copy a text file to to an object storage and assign storage-class `REDUCED_REDUNDANCY` to the uploaded object.*
+
+```sh
+mc cp -storage-class REDUCED_REDUNDANCY myobject.txt play/mybucket
```

Comment:
Should this be changed to `mc cp -storage-class REDUCED_REDUNDANCY myobject.txt play/mybucket` to `mc cp --storage-class REDUCED_REDUNDANCY myobject.txt play/mybucket` ?